### PR TITLE
Add support for specifying <title> via seoTitle

### PR DESCRIPTION
**Pull request by sebastianbenz**
Enables specifying the title tag content via an optional frontmatter property seoTag. This way document authors can provide a better title tag if the document title is not sufficient.

- Updated meta template to use `seoTitle` if present and fallback to `title`
  otherwise.
- Add explainer to template doc
- Add seoTitle for a few guides with a too generic title

Fixes https://github.com/GoogleChrome/developer.chrome.com/issues/4389

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@matthiasrohmer went with `seoTitle`. PTAL

**Review by matthiasrohmer — APPROVED** [review]
Small copy/paste error that broke the build, otherwise the template change looks good to me.

Forwarding to @rachelandrew for content approval.

**Comment by sebastianbenz** [comment]
@matthiasrohmer thanks for the fix!

**Review by devnook — APPROVED** [review]

File: site/en/docs/extensions/mv3/getstarted/development-basics/index.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: 'layouts/doc-post.njk'
 title: 'Development basics'
+seoTitle: 'Chrome Extension Development basics'
```
Comment (rachelandrew) [inline site/en/docs/extensions/mv3/getstarted/development-basics/index.md:4]:
```suggestion
seoTitle: 'Chrome Extension development basics'
```

File: site/en/docs/lighthouse/overview/index.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: 'layouts/doc-post.njk'
 title: Overview
+seoTitle: Lighthouse Overview
```
Comment (rachelandrew) [inline site/en/docs/lighthouse/overview/index.md:4]:
```suggestion
seoTitle: Lighthouse overview
```

File: site/en/docs/android/custom-tabs/low-level-api/index.md [hunk 3]
```diff
@@ -1,9 +1,10 @@
 ---
 layout: "layouts/doc-post.njk"
 title: Low level API 
+seoTitle: Custom Tabs Low level API
```
Comment (rachelandrew) [inline site/en/docs/android/custom-tabs/low-level-api/index.md:4]:
```suggestion
seoTitle: Custom Tabs low level API 
```

File: site/en/docs/android/trusted-web-activity/index.md [hunk 4]
```diff
@@ -1,9 +1,10 @@
 ---
 layout: "layouts/doc-post.njk"
 title: Overview
+seoTitle: Android Trusted Web Activities Overview
```
Comment (rachelandrew) [inline site/en/docs/android/trusted-web-activity/index.md:4]:
```suggestion
seoTitle: Android Trusted Web Activities overview
```

File: site/en/docs/extensions/mv3/a11y/index.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: "layouts/doc-post.njk"
 title: "Accessibility (a11y)"
+seoTitle: "Chrome Extensions Accessibility (a11y)"
```
Comment (rachelandrew) [inline site/en/docs/extensions/mv3/a11y/index.md:4]:
```suggestion
seoTitle: "Chrome Extensions accessibility (a11y)"
```

File: site/en/docs/extensions/mv3/architecture-overview/index.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: "layouts/doc-post.njk"
 title: "Architecture overview"
+seoTitle: "Chrome Extensions Architecture overview"
```
Comment (rachelandrew) [inline site/en/docs/extensions/mv3/architecture-overview/index.md:4]:
```suggestion
seoTitle: "Chrome Extensions architecture overview"
```

File: site/en/docs/extensions/mv3/content_scripts/index.md [hunk 7]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: "layouts/doc-post.njk"
 title: "Content scripts"
+seoTitle: "Chrome Extensions Content scripts"
```
Comment (rachelandrew) [inline site/en/docs/extensions/mv3/content_scripts/index.md:4]:
```suggestion
seoTitle: "Chrome Extensions content scripts"
```

File: site/en/docs/extensions/mv3/cross-origin-isolation/index.md [hunk 8]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: "layouts/doc-post.njk"
 title: "Cross-origin isolation"
+seoTitle: "Chrome Extensions Cross-origin isolation"
```
Comment (rachelandrew) [inline site/en/docs/extensions/mv3/cross-origin-isolation/index.md:4]:
```suggestion
seoTitle: "Chrome Extensions cross-origin isolation"
```

File: site/en/docs/extensions/mv3/getstarted/index.md [hunk 9]
```diff
@@ -1,8 +1,9 @@
 ---
 layout: 'layouts/doc-post.njk'
 title: 'Welcome'
+seoTitle: 'Chrome Extensions getting started guides '
 subhead: 'Welcome to the Getting Started Guides.'
-description: 'Welcome to the Getting Started Guides.'
+description: 'Overview of all Chrome Extensions Getting Started Guides.'
```
Comment (rachelandrew) [inline site/en/docs/extensions/mv3/getstarted/index.md:6]:
```suggestion
description: 'Overview of all Chrome Extensions getting started guides.'
```

**Review by rachelandrew — APPROVED** [review]
I just fixed casing. Please commit those before merging.

**Comment by sebastianbenz** [comment]
@rachelandrew thanks for fixing these! They're all committed. 

Could someone please trigger a build and merge (seems like I don't have permissions). Thanks!

**Pull request merged**